### PR TITLE
🌐 Handle language in qs for authenticated middleware

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: middleware/authenticated.js [hunk 1]
```diff
@@ -43,15 +43,18 @@ export default function ({
...
+    register, from, referrer, language,
+  } = query;
   if (!store.getters.getUserIsRegistered) {
     let redirectPath = '/in/register';
-    const { register, from, referrer } = query;
     if (!process.server) {
       store.commit(USER_SET_AFTER_AUTH_ROUTE, route);
       setRouteToSessionStorage(route);
-      if (register === '1') redirectPath = `${redirectPath}?register=1`;
+      if (register === '1') redirectPath = `${redirectPath}?register=1&language=${language}`;
```
Comment (williamchong) [inline middleware/authenticated.js:54]: Will language be set as `undefined` if it is not set? should we check for `language` exists or default it to `''` ?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: middleware/authenticated.js (hunk 1 above)
Comment (nwingt) [reply middleware/authenticated.js:56]: Added default

Comment (williamchong) [reply middleware/authenticated.js:56]: I think it's better to not import the default locale into this file. Can we not handle language if it is not set in qs, and focus the handling of default locale in the i18n plugin file?

Comment (nwingt) [reply middleware/authenticated.js]: Updated

**Review by williamchong — APPROVED** [review]
Hmm not sure why the old code specially split the case to only include all qs in `process.server`. But if it works fine including all qs in redirect should not be an issue

**Pull request merged**